drivers/crypto/CMakeLists.txt

```diff
@@ -1,20 +1,25 @@
 # SPDX-License-Identifier: Apache-2.0
 
 zephyr_library()
-zephyr_library_sources_ifdef(CONFIG_CRYPTO_ATAES132A		crypto_ataes132a.c)
-zephyr_library_sources_ifdef(CONFIG_CRYPTO_MBEDTLS_SHIM		crypto_mtls_shim.c)
-zephyr_library_sources_ifdef(CONFIG_CRYPTO_STM32			crypto_stm32.c)
-zephyr_library_sources_ifdef(CONFIG_CRYPTO_STM32_HASH		crypto_stm32_hash.c)
-zephyr_library_sources_ifdef(CONFIG_CRYPTO_SMARTBOND        crypto_smartbond.c)
-zephyr_library_sources_ifdef(CONFIG_CRYPTO_NRF_ECB		crypto_nrf_ecb.c)
-zephyr_library_sources_ifdef(CONFIG_CRYPTO_INTEL_SHA		crypto_intel_sha.c)
-zephyr_library_sources_ifdef(CONFIG_CRYPTO_NPCX_SHA		crypto_npcx_sha.c)
-zephyr_library_sources_ifdef(CONFIG_CRYPTO_MCHP_XEC_SYMCR	crypto_mchp_xec_symcr.c)
+
+# zephyr-keep-sorted-start
+zephyr_library_sources_ifdef(CONFIG_CRYPTO_ATAES132A            crypto_ataes132a.c)
+zephyr_library_sources_ifdef(CONFIG_CRYPTO_CC23X0               crypto_cc23x0.c)
+zephyr_library_sources_ifdef(CONFIG_CRYPTO_INTEL_SHA            crypto_intel_sha.c)
 zephyr_library_sources_ifdef(CONFIG_CRYPTO_IT51XXX_SHA		crypto_it51xxx_sha.c)
```

Comment:
Despite all other entries being transitioned to uniform space indentation, this remains tabs